From ad4bd28c2dd6c066b7eabcacb9a7e43fd9a14d29 Mon Sep 17 00:00:00 2001
From: Alejandro Sierra <alejandro.sierra@nxp.com>
Date: Mon, 26 Sep 2016 14:58:36 -0500
Subject: [PATCH 1254/5242] MLK-13286 ARM: dts: imx: support emmc on 6sxscm
 platforms

commit  7084596e5da8fd47e9fef7c95be810168d4bb5d7 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add the generic dtsi configuration to support EMMC on the
i.MX6SX SCM platforms

Signed-off-by: Alejandro Sierra <alejandro.sierra@nxp.com>
Signed-off-by: Juan Gutierrez <juan.gutierrez@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sxscm-emmc.dtsi |   21 +++++++++++++++++++++
 1 file changed, 21 insertions(+)
 create mode 100644 arch/arm/boot/dts/imx6sxscm-emmc.dtsi

diff --git a/arch/arm/boot/dts/imx6sxscm-emmc.dtsi b/arch/arm/boot/dts/imx6sxscm-emmc.dtsi
new file mode 100644
index 0000000..7e616f4
--- /dev/null
+++ b/arch/arm/boot/dts/imx6sxscm-emmc.dtsi
@@ -0,0 +1,21 @@
+/*
+ * Copyright (C) 2016 Freescale Semiconductor, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+&usdhc4 {
+	pinctrl-names = "default", "state_100mhz", "state_200mhz";
+	pinctrl-0 = <&pinctrl_usdhc4_1>;
+	pinctrl-1 = <&pinctrl_usdhc4_1_100mhz>;
+	pinctrl-2 = <&pinctrl_usdhc4_1_200mhz>;
+	bus-width = <8>;
+	no-1-8-v;
+	non-removable;
+	status = "okay";
+};
+
+&spdif {
+	status = "disabled";
+};
-- 
1.7.9.5

